main.go

```diff
@@ -64,12 +83,12 @@ func main() {
 	if parsedArgs.Memprofile != "" {
 		f, err := os.Create(parsedArgs.Memprofile)
 		if err != nil {
-			log.Fatal("could not create memory profile: ", err)
+			log.Fatal().Msg("could not create memory profile: ")
 		}
 		defer f.Close() // error handling omitted for example
 		//runtime.GC()    // get up-to-date statistics
 		if err := pprof.WriteHeapProfile(f); err != nil {
-			log.Fatal("could not write memory profile: ", err)
+			log.Fatal().Msg("could not write memory profile: ")
```

Comment:
same comment as previous:
```suggestion
			log.Fatal().Err(err).Msg("could not write memory profile: ")
```